<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QMainWindow" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.49126, y1:0.046, x2:0.505753, y2:1, stop:0.301948 rgba(0, 118, 212, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_avatar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>24</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/avatar.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>371</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_lock">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>24</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../PyQt5/projetos/SistemaDeLogin/padlock.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>371</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_profile">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:65%;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/profile.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRemember">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Remember me</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbIncorrect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 132, 0);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Register ?</string>
    </property>
   </widget>
   <zorder>input_password</zorder>
   <zorder>input_username</zorder>
   <zorder>img_avatar</zorder>
   <zorder>img_lock</zorder>
   <zorder>img_profile</zorder>
   <zorder>btnLogin</zorder>
   <zorder>cbRemember</zorder>
   <zorder>lbIncorrect</zorder>
   <zorder>btnRegister</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:white;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRegister"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
